position_controllers/src/joint_group_position_controller.cpp

```diff
@@ -42,10 +42,10 @@ template <class T>
 void forward_command_controller::ForwardJointGroupCommandController<T>::starting(const ros::Time& time)
 {
   // Start controller with current joint positions
-  commands_.resize(n_joints_, 0.0);
+  std::vector<double> & commands = *commands_buffer_.readFromRT();
```

Comment:
same thing I mention above about writing into a shared resource without synchronization primitives in place.